app/sidekiq/add_user_to_department_job.rb

```diff
@@ -0,0 +1,35 @@
+class AddUserToDepartmentJob
```

Comment:
nêu chạy sidekiq background job thì dùng worker dc k?
dùng sidekiq worker thì khác gì vs job?

Reply:
Có thể sử dụng ActiveJob được cung cấp mặc định bởi Rails để chạy Job tuy nhiên Sidekiq có một số lợi thế khi đem ra so sánh như trong khi ActiveJob lưu trực tiếp các Job trong RAM, nếu server bị shutdown nó sẽ bị mất hết các job trong khi Sidekiq sử dụng Redis để lưu các Job bởi vậy nó an toàn hơn trong việc toàn vẹn dữ liệu.

Reply:
ready